Makefile

```diff
@@ -24,7 +24,7 @@ cover: test  ## Show test coverage in your browser
 clean::
 	rm -f $(COVERFILE)
 
-CHECK_COVERAGE = awk -F '[ \t%]+' '/^total:/ && $$3 < $(COVERAGE) {exit 1}'
+CHECK_COVERAGE = awk -F '[ \t%]+' '/^total:/ {print; if ($$3 < $(COVERAGE)) exit 1}'
```

Comment:
Technically, this is "Print coverage on all checks", not just failed. Aren't I such a nit-picker?